src/ert/config/analysis_module.py

```diff
@@ -223,7 +217,7 @@ def set_var(self, var_name: str, value: Union[float, int, bool, str]) -> None:
                     value = str(value).lower() != "false"
 
                 var["value"] = var["type"](value)
-        else:
+        elif var_name not in self.JUST_REMOVE_KEYWORDS:
```

Comment:
Is there a conditional branch where `var_name` is in `self.JUST_REMOVE_KEYWORDS` is handled?  Also as far as I can tell now unknown `var_names` will no longer throw a config validation error. Is this intended behaviour?

Reply:
It's actually reverse. We don't raise if the keyword is in that list, otherwise (for all unknown `var_names`) we raise the error.
The keywords found in `self.JUST_REMOVE_KEYWORDS` have are already been suggested in the UI at this point, and should be disregarded since they have no effect.

I can take a look and see if I can do this in a more readable way?

Reply:
```
        elif var_name in self.JUST_REMOVE_KEYWORDS:
            pass
        else:
            raise ConfigValidationError(
                f"Variable {var_name!r} not found in {self.name!r} analysis module"
            )
```
I guess this is might be more readable, avoiding the `not` in there?